File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int size = 0;
    do
    {
        size = get_int("Height: ");
    }
    while (!(size > 0 && size < 9));


    for (int i = 1; i <= size; i++)
    {
        for (int j = size; j > 0; j--)
        {
            if (j <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int


height = 0
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

for i in range(1, height+1):
    for j in range(height, 0, -1):
        if j <= i:
            print("#", end="")
        else:
            print(" ", end="")
    print()
